Neos.Media.Browser/Resources/Private/Partials/ContentDefaultPreview.html

```diff
@@ -1,7 +1,14 @@
 {namespace m=Neos\Media\ViewHelpers}
 {namespace neos=Neos\Neos\ViewHelpers}
 <div class="neos-preview-image" id="neos-preview-image">
-    <a href="{assetProxy.originalUri}" target="_blank">
-        <img src="{assetProxy.previewUri}" class="img-polaroid" alt="{assetProxy.label}"/>
-    </a>
+    <f:if condition="{assetContainsMaliciousContent}">
+        <f:then>
+            <img src="{assetProxy.previewUri}" class="img-polaroid" alt="{assetProxy.label}"/>
```

Comment:
Are we sure the `previewUri` does never point to the potentially malicious content? 🤔

Reply:
No, we don't. But the code is just excecuted if you open the file in the browser directly. It's not executed if you embed it in an img-tag.